packages/cozy-stack-client/src/CozyStackClient.js

```diff
@@ -99,6 +130,20 @@ class CozyStackClient {
    * @throws {FetchError}
    */
   async fetchJSON(method, path, body, options = {}) {
+    try {
+      return await this._fetchJSON(method, path, body, options)
+    } catch (e) {
+      if (/Expired token/.test(e.message)) {
```

Comment:
Is there no other way than depending on the error message?

Reply:
J'avoue que c'est du code présent ailleurs que j'ai déplacé sans y toucher.

L'erreur vient de https://github.com/cozy/cozy-stack/blob/745bb33ab8163c0d7dab7525068a3b4b48ad9382/pkg/permissions/errors.go#L21 

À priori si on veut faire plus il faudra ajouter une notion de code d'erreur côté stack. Ca me semble toutefois assez stable ainsi. Tu préférerais qu'on ajoute un système de code d'erreur côté stack ?

Reply:
:ok_hand: Maintenant que tu le dis, je crois qu'on a été obligé de faire pareil côté desktop.
J'avoue qu'une détection d'erreur basée sur le message allume une (petite) lampe chez moi, je trouverais bien de pouvoir faire autrement, mais je ne sais pas dans quelle mesure c'est viable, et ça n'a évidemment ni à bloquer ta PR, ni à être traité en priorité côté stack.